elastic/integrations
# [New Integration] Goflow2 integration

The GoFlow2 integration allows you to monitor goflow2 logs. At the moment only goflow2 sflow logs are supported.

## Checklist



## How to test this PR locally

- elastic-package check
- elastic-package install
- install elatic-agent
- generate goflow2 sflow logs
  -  `goflow2 -format json -listen "sflow://:6343" -mapping /root/sflow/goflow2/mapping.yaml -transport.file /var/log/sflow/goflow2/goflow2.log`

```
# File: mapping.yaml
formatter:
    fields: # list of fields to format in JSON
        - type
        - time_flow_start_ns
        - sampler_address
        - sequence_num
        - in_if
        - out_if
        - src_addr
        - dst_addr
        - etype
        - proto
        - src_port
        - dst_port
        - src_vlan
        - dst_vlan
        - ip_flags
        - sampling_rate
        - bytes
```
